aries-site/src/layouts/main/Layout.js

```diff
@@ -53,58 +58,79 @@ export const Layout = ({
       {/* When a backgroundImage is present, the main page content becomes 
       the `last` child. We want this content to drive the layout.
       For details on this prop, see here: https://v2.grommet.io/stack#guidingChild */}
-      <Stack fill guidingChild={backgroundImage && 'last'}>
-        {backgroundImage && (
-          <PageBackground backgroundImage={backgroundImage} />
-        )}
-        <Page>
-          {/* I think Head is redundant at this point, 
+      <ResponsiveContext.Consumer>
+        {size => (
+          <Stack fill guidingChild={backgroundImage && 'last'}>
+            {backgroundImage && (
+              <PageBackground backgroundImage={backgroundImage} />
+            )}
+            <Page>
+              {/* I think Head is redundant at this point, 
               but left it as is for now */}
-          <Head title={title} />
-          <Meta
-            title={title}
-            render={render}
-            description={seoDescription}
-            canonicalUrl={`https://design-system.hpe.design${router.route}`}
-          />
-          <>
-            <SkipLinks id="skip-links">
-              <SkipLink id="main" label="Main Content" />
-            </SkipLinks>
-            <PageContent>
-              <Header fill="horizontal" alignSelf="center" />
-            </PageContent>
-            <MainContentWrapper>
-              <Main overflow="visible">
-                <SkipLinkTarget id="main" />
-                <Box>
-                  {layout !== 'plain' ? (
-                    <>
-                      <ContentSection>
-                        <DocsPageHeader
-                          title={title}
-                          topic={topic}
-                          render={render}
-                        />
-                        {children}
-                      </ContentSection>
-                      {relatedContent.length > 0 && (
-                        <RelatedContent
-                          relatedContent={relatedContent}
-                          title={title}
-                        />
+              <Head title={title} />
+              <Meta
+                title={title}
+                render={render}
+                description={seoDescription}
+                canonicalUrl={`https://design-system.hpe.design${router.route}`}
+              />
+              <>
+                <SkipLinks id="skip-links">
+                  <SkipLink id="main" label="Main Content" />
+                </SkipLinks>
+                <PageContent>
+                  <Header fill="horizontal" alignSelf="center" />
+                </PageContent>
+                <MainContentWrapper>
```

Comment:
This page is rendering a bit weird -- not sure exactly what's driving it but putting the comment here: https://deploy-preview-2753--keen-mayer-a86c8b.netlify.app/templates/content-layouts

<img width="1677" alt="Screen Shot 2022-07-25 at 4 30 30 PM" src="https://user-images.githubusercontent.com/12522275/180891700-78b0541b-f74e-4a20-bab7-c8c1aed7f927.png">

Reply:
notice how content is being crushed / overlapped.

Reply:
This one as well: https://deploy-preview-2753--keen-mayer-a86c8b.netlify.app/templates/navigation